### PR TITLE
add cerb changes to build proper

### PR DESCRIPTION
**Pull request by rubennoro**
added functions for stm and added the global i2c_handler to intiailize pca structs properly

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

File: Core/Src/pdu.c [hunk 1]
```diff
@@ -19,6 +19,30 @@
 
 static osMutexAttr_t pdu_mutex_attributes;
 
+//hi2c2 variable to pass to the function wrappers
+extern I2C_HandleTypeDef hi2c2;
```
Comment (jr1221) [inline Core/Src/pdu.c:23]: @caiodasilva2005 Lets either use externs or pass by pointer, not to both.  So whatever you like more, remove the other.  If you like externs then we should remove the i2c from the `pdu` struct or if you are an extern hater (like me but ignore my opinion) then we should pass the i2c struct into the pca_i2c_write somehow idk how.
